src/sgame/sg_spawn_mover.cpp

```diff
@@ -594,14 +594,41 @@ bool IsAutomaticMover( const gentity_t *ent )
 	return true;
 }
 
-void BotHandleDoor( gentity_t *ent )
+// This function adds obstacles for doors that the bot can't open,
+// that is doors that are not automatic, or doors that are already opened.
```

Comment:
Why are open doors obstacles?

Reply:
Open door are obstacles because they are not going to open more if you get closer, so you need to avoid walking on the door pane if it's sticking out of the wall. See also https://github.com/Unvanquished/Unvanquished/issues/2358.

Reply:
Ah interesting.